### ISSUE TITLE
[Legitimate Site Blocked] bakedbeans.io

### ISSUE THREAD

**Issue by mdixon888** [MetaMask/eth-phishing-detect#9083]
https://bakedbeans.io/?ref=0xa9FfaBA9Cc51b7344BeF7d7abc394D516cCc4e86

This is a legitimate site. I've been using it for 6 months! As it's been blocked I cannot access my coins, which is very annoying. Please unblock!

**Comment by 409H** [comment]
Closing. Duplicate of https://github.com/MetaMask/eth-phishing-detect/issues/9062
